src/kili/graphql_client.py

```diff
@@ -42,7 +42,7 @@ def __init__(
         self.token = None
         self.verify = verify
 
-    def execute(self, query, variables=None):
+    def execute(self, query, variables=None) -> dict:
```

Comment:
`dict`-> `Dict` , here and below. We use Python types and not the native structures for typing, to stay consistent.

Reply:
(it may be not the case everywhere in the existing code, but that's a mistake)